src/Ledger/Ratify.lagda

```diff
@@ -201,16 +201,16 @@ mostStakeDRepDist-∅ {dist} = suc (Σᵐᵛ[ x ← dist ᶠᵐ ] x) , Propertie
         <⟨ v>sum ⟩
       v ∎
 
-∃topNDRepDist : ∀ {n dist} → lengthˢ (dist ˢ) ≥ n → n > 0
-                → ∃[ c ] lengthˢ (mostStakeDRepDist dist c ˢ) ≥ n
+∃topNDRepDist : ∀ {n dist} → n ≤ lengthˢ (dist ˢ) → n > 0
```

Comment:
Reserve `≥` for rationals in this module.

(quick and dirty way to use `≥` in place of `ℚ.≥` below)

Reply:
Now that you worked so hard on the typeclasses, you should just make an instance for `ℚ`!